enterprise-suite/tests/resources/alert-app.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: es-alert-test
+spec:
+  replicas: 2
```

Comment:
could be 1 replica?

Reply:
Sure.  Will change.  I just blindly copied `app.yaml`.